README.md

```diff
@@ -1,4 +1,4 @@
-# IRremoteESP8266 Library
+![IRremoteESP8266 Banner](https://user-images.githubusercontent.com/11258286/103334782-b52c5d00-4a98-11eb-89c1-33671e2a9720.png)
```

Comment:
Change "Banner" to "Library". That way the alt text is still okay for people who don't have images or are vision impaired.

+1 to @NiKiZe comment about the image being self hosted in the repo, and if an svg is an option.

As he has better insight in this matter, I'm going to leave him to do the final approval.